Makefile

```diff
@@ -105,10 +114,41 @@ docker_bats_parallel:
 		${DOCKER_IMAGE_BATS}:${DOCKER_TAG_BATS} \
 		/bin/bash -c "terraform init && time find test/spec/ -name '*.bats' -print0 | xargs -0 -P4 --no-run-if-empty -n1 bats"
 
-# Run docker bats shell
-.PHONY: docker_bats_shell
-docker_bats_shell:
+## BEGIN Resources
+# Model resources in the CI Pipeline definition. For example,
+# integration_test_image models the integration-test-image resource in CI.
+
+# Fetch the image.
+.PHONY: integration_test_image
+integration_test_image:
+	docker pull ${DOCKER_IMAGE_INTEGRATION_URI}
+## END Resources
+
+## BEGIN Job Tasks
+# Model the CI integration-tests job definition and tasks as a make target.
+# The Git repo is placed in the same path the CI pull-request resource will be
+# placed.  Params are passed in the same way params are passed in the CI run
+#
+# Unlike other modules which use a *.tfvars file, this module uses environment
+# variables to pass inputs into the integration tests.  Pass along the same
+# variables set as parameters of the CI Job.
+.PHONY: integration_test_run
+integration_test_run: integration_test_image
```

Comment:
-1: We have [prior art](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/Makefile#L150) for the name of this make target, for consistency's sake I think we should either rename this to `test_integration_docker` or rename the Project Factory target to `test_integration_run`. Either is good by me, but let's use this chance to pick a standard name.

Reply:
Fixed in 6039656